requirements.txt

```diff
@@ -2,4 +2,4 @@ click
 progress
 pyyaml
 simplejson
-cryptography
+cryptography==3.4
```

Comment:
Why pin it at such an ancient version? Upstream is at 36.0.2. If there is a reason, could you include it in the commit message and requirements.txt?

Reply:
Hi @nicolasbock, you can see it in the commit: https://github.com/canonical/hotsos/pull/328/commits/1bf39c8a7518c996535016c65d8f7394e9300243 and in the description of the PR as well.

Is there anything I am missing?

Reply:
Oh, i'll add a comment on requirements.txt

Reply:
> Is there anything I am missing?

Yes, the why :)

This version might be great, I was just wondering why you pin it to a release that is from February 7 2021.

Reply:
@nicolasbock fixed on https://github.com/canonical/hotsos/commit/2dcc79dd9a0acf20de1dd24ef48fdb60441f4275 LMK if it is ok like that or you want more info.
OTOH I would also like to be able to build the snap with `cryptography` with the greatest and latest, but this was the quick fix I could find in order for us to not lose sight on when a certificate is expiring. 
I can create a new issue to track this work. WDYT?

Reply:
@nicolasbock fixed on https://github.com/canonical/hotsos/commit/2dcc79dd9a0acf20de1dd24ef48fdb60441f4275 LMK if it is ok like that or you want more info.
OTOH I would also like to be able to build the snap with `cryptography` with the greatest and latest, but this was the quick fix I could find in order for us to not lose sight on when a certificate is expiring. 
I can create a new issue to track this work. WDYT?

Reply:
Does this build with `3.4.8`? That's the last version in the `3.4.x` branch.

Reply:
@nicolasbock good point, yes it does. I've just tested it and pushed the deps with 3.4.8 on https://github.com/canonical/hotsos/commit/b3c6ff910859269170c5997b7fe143fdb0d8fc4e